src/version/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod block;
```

Comment:
this module got moved into the wranglerjs module in a previous commit; we should be able to remove this and just have  `src/version.rs`